HPXMLtoOpenStudio/measure.rb

```diff
@@ -612,6 +631,12 @@ def self.update_solar_absorptances(runner, model)
       innermost_material = layered_const.layers[layered_const.numLayers() - 1].to_StandardOpaqueMaterial.get
       innermost_material.setSolarAbsorptance(0.0)
       innermost_material.setVisibleAbsorptance(0.0)
+      next unless not adj_surface.nil?
+      # Create new construction in case of shared construciton.
+      layered_const_adj = OpenStudio::Model::Construction.new(model)
+      layered_const_adj.setName(cond_bsmnt_surface.construction.get.name.get + ' Reversed Bsmnt')
+      adj_surface.setConstruction(layered_const_adj)
+      layered_const_adj.setLayers(cond_bsmnt_surface.construction.get.to_LayeredConstruction.get.layers.reverse())
```

Comment:
I modified the adjacent surface construction based on the new surface construction, the FT errors are gone. But not sure whether this makes sense in terms of modeling.

Reply:
It seems reasonable to me.